src/mixxx.cpp

```diff
@@ -1229,6 +1229,13 @@ void MixxxMainWindow::initActions()
     m_pHelpManual->setWhatsThis(buildWhatsThis(manualTitle, manualText));
     connect(m_pHelpManual, SIGNAL(triggered()), this, SLOT(slotHelpManual()));
 
+    QString shortcutsTitle = tr("&Keyboard Shortcuts");
```

Comment:
Agree that we should indicate outbound links in the help menu.
This would not only affect this very link, but all other external links in the help menu. We also had to distinguish between the kind of links for the manual ( Win/Linux= local PDF, OSX= online)

Using `...` to indicate outbound links might be not universal understandable, in the manual we use an icon suffix, alternatively i`d prefer a suffix like ``(Online)``